spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -95,42 +121,42 @@ interface ErrorReceipt {
 
 ### Store Paths
 
-#### Upgrade Sequence Path
+#### Upgrade Channel Path
 
-The upgrade sequence path is a public path that stores the current sequence of the upgrade attempt. The sequence will increment with each attempted upgrade on the given channel. The sequence will be used to ensure that different error receipts referring to different upgrade attempts do not interfere with each other.
+The chain must store the proposed upgrade until the upgrade handshake successfully. This must be stored in the publice store. It may be deleted once the upgrade is successful.
 
 ```typescript
-function channelUpgradeSequencePath(portIdentifier: Identifier, channelIdentifier: Identifier) Path {
-    return "channelUpgrades/upgradeSequence/ports/{portIdentifier}/channels/{channelIdentifier}"
-}
+function channelUpgradePath(portIdentifier: Identifier, channelIdentifier: Identifier): Path {
+    return "channelUpgrades/upgrades/ports/{portIdentifier}/channels/{channelIdentifier}"
+ }
 ```
 
-The upgrade sequence MUST also have a verification method so that chains can prove the upgrade sequence on the counterparty for the given channel upgrade.
+The upgrade path has an associated verification membership added to the connection interface so that a counterparty may verify that chain has stored the provided counterparty upgrade.
```

Comment:
```suggestion
The upgrade path has an associated verification of membership method added to the connection interface so that a counterparty may verify that chain has stored the provided counterparty upgrade.
```
any wording is fine, but "verification membership" didn't read well to me